models/Element/ElementInterface.php

```diff
@@ -61,7 +61,8 @@ public function getUserModification(): ?int;
 
     public function setUserModification(int $userModification): static;
 
-    public static function getById(int $id): ?static;
+    //TODO add $params parameter in Pimcore 12
+    public static function getById(int $id /* array $params = [] */): ?static;
```

Comment:
```suggestion
    public static function getById(int $id /*, array $params = [] */): ?static;
```

Reply:
Sorry for that, I noticed that the commit message got messed up this time so I corrected it and force pushed